use-package.el

```diff
@@ -168,7 +168,7 @@ convert it to a string and return that."
 (defun use-package-load-name (name &optional noerror)
   "Return a form which will load or require NAME depending on
 whether it's a string or symbol."
-  (if (stringp name) `(load ,name 'noerror) `(require ',name nil 'noerror)))
```

Comment:
Oh, I made a mistake in #225, those `'noerror`s should have been `,noerror`. You should probably drop the `noerror` param if you're going to ignore it anyway.